pages/index.js

```diff
@@ -398,94 +398,10 @@ export default function Home() {
-                          Course Name
-                        </th>
-                        <th
-                          scope="col"
-                          className="p-2 md:px-6 md:py-3 text-left uppercase tracking-wider"
-                        >
-                          <span className="hidden lg:block">
-                            AICTE approved course fee per student (in Rs.)
-                          </span>
-                          <span className="block lg:hidden">
-                            course fee per student (in Rs.)
-                          </span>
-                        </th>
-                      </tr>
-                    </thead>
-                    <tbody className="bg-gradient-to-r w-full from-gray-800 to-gray-900 text-gray-50">
-                      <tr className="font-mono bg-gray-700 bg-opacity-50 text-sm md:text-lg border border-gray-500">
-                        <td className="p-2 md:p-6 lg:whitespace-nowrap font-medium ">
-                          1
-                        </td>
-                        <td className="p-2 md:p-6 lg:whitespace-nowrap border-r border-gray-500">
-                          Web Development 101 (Getting Started with JavaScript)
-                        </td>
-                        <td
-                          className="p-2 md:p-6 lg:whitespace-nowrap border-r border-gray-500"
-                          rowspan="2"
-                        >
-                          10,000
-                        </td>
-                      </tr>
-
-                      <tr className="font-mono bg-gray-700 bg-opacity-50 text-sm md:text-lg border border-gray-500">
-                        <td className="p-2 md:p-6 lg:whitespace-nowrap font-medium ">
-                          2
-                        </td>
-                        <td className="p-2 md:p-6 lg:whitespace-nowrap border-r border-gray-500">
-                          Web Development 201 (Server-side programming with Ruby
-                          on Rails)
-                        </td>
-                      </tr>
-
-                      <tr className="font-mono text-sm md:text-lg">
-                        <td className="p-2 md:p-6 lg:whitespace-nowrap font-medium">
-                          3
-                        </td>
-                        <td className="p-2 md:p-6 lg:whitespace-nowrap">
-                          Web Development 301 (Typed Functional Programming)
-                        </td>
-                        <td className="p-2 md:p-6 lg:whitespace-nowrap">
-                          10,000
-                        </td>
-                      </tr>
-                    </tbody>
-                  </table>
-                </div>
-              </div>
             </div>
           </div>
         </section>
-%Reena
+
 
 <section className="bg-gray-800 border-t border-b border-gray-700 mx-auto relative">
           <div className="max-w-6xl 2xl:max-w-7xl mx-auto">
```

Comment:
@reenas 
`ml-8 max-w-4xl font-mono text-white tracking-tight pt-1 lg:text-lg` 
Please remove `ml-8` class from the price details paragraph.

Also remove `md:ml-8` class from `sm:rounded-lg md:ml-8 mt-8`.